doc/pcre2syntax.3

```diff
@@ -627,22 +627,26 @@ following ways:
   (?(condition)yes-pattern)
   (?(condition)yes-pattern|no-pattern)
 .sp
-  (?(n)               absolute reference condition
-  (?(+n)              relative reference condition (PCRE2 extension)
-  (?(-n)              relative reference condition (PCRE2 extension)
-  (?(<name>)          named reference condition (Perl)
-  (?('name')          named reference condition (Perl)
-  (?(name)            named reference condition (PCRE2, deprecated)
-  (?(R)               overall recursion condition
-  (?(Rn)              specific numbered group recursion condition
-  (?(R&name)          specific named group recursion condition
-  (?(DEFINE)          define groups for reference
-  (?(VERSION[>]=n.m)  test PCRE2 version
-  (?(assert)          assertion condition
+  (?(n)                absolute reference condition
+  (?(+n)               relative reference condition (PCRE2 extension)
+  (?(-n)               relative reference condition (PCRE2 extension)
+  (?(<name>)           named reference condition (Perl)
+  (?('name')           named reference condition (Perl)
+  (?(name)             named reference condition (PCRE2, deprecated)
+  (?(R)                overall recursion condition
+  (?(Rn)               specific numbered group recursion condition
+  (?(R&name)           specific named group recursion condition
+  (?(DEFINE)           define groups for reference
+  (?(VERSION[>]=n[.m]) test PCRE2 version
+  (?(assert)           assertion condition
 .sp
 Note the ambiguity of (?(R) and (?(Rn) which might be named reference
 conditions or recursion tests. Such a condition is interpreted as a reference
 condition if the relevant named group exists.
+.sp
+The parts within brackets for the VERSION conditional syntax could be ommited.
+The fractional part of the version number defaults to 0, and can be up to two
+digits long.
```

Comment:
Where do you get "can be up to two digits long"? It appears to support up to `999.999` (big enough for any version we'll ever use - and so probably no need to specify what the limit is).

Reply:
But good spot that the fractional part is optional! Nice doc improvement.